### PR TITLE
error: use http-custom-errors

### PR DESCRIPTION
**Pull request by yoshuawuyts**
## Changes
- **error**: provide formatted errors using `http-custom-errors` so boilerplate can be reduced

### CODE REVIEW

**Comment by yoshuawuyts** [comment]
cc @TabDigital/ping-api

File: index.js [hunk 1]
```diff
@@ -38,5 +39,5 @@ function nodeMiddlewareJsonParse (ctx, propName, fmtErr) {
 // default JSON format fn
 // null -> obj
 function dftFmt () {
-  return { message: 'Invalid JSON' }
+  return new httpError.BadRequestError('Invalid JSON')
```
Comment (kiddkai) [inline index.js:42]: AWESOME!!!

**Comment by kiddkai** [comment]
:+1:

**Comment by yoshuawuyts** [comment]
@rprieto can you publish this as a patch version? Thanks!

**Pull request merged**